.travis.yml

```diff
@@ -10,35 +10,46 @@ before_install:
   # Update conda itself
   - conda update --yes conda
 install:
-  # At this moment this plugin has overlapping dependencies than qiita so installing everything in the same environment
-  - conda create --yes -n qiita_env python=2.7 pip nose flake8
+  # installing qiita environment
+  - conda create --yes -n qiita python=2.7 pip nose flake8
     pyzmq networkx pyparsing natsort mock future libgfortran
     'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1' matplotlib
-  - source activate qiita_env
-  - pip install sphinx sphinx-bootstrap-theme coveralls ipython[all]==2.4.1
+  - source activate qiita
+  - pip install sphinx sphinx-bootstrap-theme coveralls
   - pip install https://github.com/biocore/qiita/archive/master.zip --process-dependency-links
-  # installing other deps
-  - conda install --yes -c jjhelmus glpk
-  - conda install --yes -c bioconda diamond bowtie2 trimmomatic=0.36 fastqc=0.11.5
-  #
-  - export MOI_CONFIG_FP=$PWD/miniconda3/envs/qiita_env/lib/python2.7/site-packages/qiita_core/support_files/config_test.cfg
-  - ipython profile create qiita-general --parallel
-  - qiita-env start_cluster qiita-general
+  - sed 's/PORT = 7777/PORT = 6379/g' ${PWD}/miniconda3/envs/qiita/lib/python2.7/site-packages/qiita_core/support_files/config_test.cfg > config_test.cfg
```

Comment:
Is this a travis-specific port conflict?

Reply:
From my experience, 6379 is the default port for redis server.

Reply:
right! the qiita main travis install has 2 redis servers (main: 7777 and redbiom: default) so here we are copying the config file and making the main qiita redis server 7777